### PR TITLE
fix documentation for import tool commands

### PR DESCRIPTION
**Pull request by srbaker**

### CODE REVIEW

File: manual/import-tool/src/docs/ops/import-tool.asciidoc [hunk 1]
```diff
@@ -87,11 +87,13 @@ We also need to reference that id space in our relationships file i.e. `START_ID
 
 === Linux
 
-Under Unix/Linux/OSX, the command is named `neo4j-import`. Depending on the installation type, the tool is either available globally, or used by executing `./bin/neo4j-import` from inside the installation directory.
+Under Unix/Linux/OSX, the command is named `neo4j-admin import --mode=csv`. Depending on the installation type, the tool
+ is either
+available globally, or used by executing `./bin/neo4j-admin import --mode=csv` from inside the installation directory.
```
Comment (benbc) [inline manual/import-tool/src/docs/ops/import-tool.asciidoc]: Our standard for docs is to keep each sentence on its own line to make diffs easier to read.

File: manual/import-tool/src/docs/ops/import-tool.asciidoc [hunk 2]
```diff
@@ -87,11 +87,13 @@ We also need to reference that id space in our relationships file i.e. `START_ID
...
 
 === Windows
 
-Under Windows, used by executing `bin\neo4j-import` from inside the installation directory.
+Under Windows, used by executing `bin\neo4j-admin import --mode=csv` from inside the installation directory.
```
Comment (benbc) [inline manual/import-tool/src/docs/ops/import-tool.asciidoc]: Can remove the mode if it's the default.

File: manual/import-tool/src/docs/ops/import-tool.asciidoc [hunk 3]
```diff
@@ -7,7 +7,7 @@ This chapter explains how to use the tool, format the input data and concludes w
```
Comment (benbc) [inline manual/import-tool/src/docs/ops/import-tool.asciidoc:7]: I suspect this entire file may be obsolete and duplicated in the docs codebase.

File: community/import-tool/src/main/java/org/neo4j/tooling/ImportTool.java [hunk 4]
```diff
@@ -104,6 +104,9 @@
         STORE_DIR( "into", null,
                 "<store-dir>",
                 "Database directory to import into. " + "Must not contain existing database." ),
+        DB_DIR( "database", null,
+                "<database-name>",
+                "Database name to import into. " + "Must not contain existing database.", true ),
```
Comment (benbc) [inline community/import-tool/src/main/java/org/neo4j/tooling/ImportTool.java]: `DB_NAME`?

File: manual/import-tool/src/test/java/org/neo4j/tooling/ImportToolDocIT.java [hunk 5]
```diff
@@ -696,10 +693,9 @@ private void printCommandToFile( String[] arguments, String dir, String fileName
             }
         }
         String documentationArgs = StringUtils.join( cleanedArguments, " " );
-        documentationArgs =
-                documentationArgs.replace( dir + File.separator, "" ).replace(
-                        directory.absolutePath().getAbsolutePath(), "path_to_target_directory" );
-        String docsCommand = "neo4j-import " + documentationArgs;
+        documentationArgs = documentationArgs.replace( dir + File.separator, "" )
+                .replace( directory.absolutePath().getAbsolutePath(), "path_to_target_directory" );
+        String docsCommand = "neo4j-admin import --mode=csv" + documentationArgs;
```
Comment (benbc) [inline manual/import-tool/src/test/java/org/neo4j/tooling/ImportToolDocIT.java]: Aren't you specifying `--mode=csv` in the individual tests? Maybe you can remove it there.

**Review by benbc — COMMENTED** [review]
Generally looks fine. Just a few comments.

File: community/import-tool/src/main/java/org/neo4j/tooling/ImportTool.java [hunk 6]
```diff
@@ -169,7 +174,7 @@
                         + ". There is a certain amount of minimum threads needed so for that reason there "
                         + "is no lower bound for this value. For optimal performance this value shouldn't be "
                         + "greater than the number of available processors." ),
-        STACKTRACE( "stacktrace", null,
+        STACKTRACE( "stacktrace", false,
                 "<true/false>",
                 "Enable printing of error stack traces." ),
         BAD_TOLERANCE( "bad-tolerance", 1000,
```
Comment (jjaderberg) [inline community/import-tool/src/main/java/org/neo4j/tooling/ImportTool.java:181]: At `CsvImporter.java:125` you change the default bad tolerance to 10 000, here it's 1000.

File: manual/import-tool/src/docs/ops/import-tool.asciidoc (hunk 3 above)
Comment (jjaderberg) [reply manual/import-tool/src/docs/ops/import-tool.asciidoc:7]: The import documentation is branched downstream. For this particular case, I think the best thing is to just apply the relevant changes upstream and create a card on our wall to make sure they are accounted for downstream.

**Pull request merged**